frontend/src/metabase/public/components/EmbedModal/EmbedModal.tsx

```diff
@@ -70,29 +50,20 @@ export const EmbedModal = ({ children, isOpen, onClose }: EmbedModalProps) => {
     setEmbedType(null);
   };
 
-  const modalHeaderProps =
-    embedType === null
-      ? {
-          label: t`Embed ${applicationName}`,
-          onClick: undefined,
-        }
-      : {
-          label: t`Static embedding`,
-          onClick: goBackToEmbedModal,
-        };
-
   return (
     <Modal
       isOpen={isOpen}
       onClose={onEmbedClose}
-      title={<EmbedTitle {...modalHeaderProps} />}
+      title={isEmbeddingSetupStage ? t`Embed ${applicationName}` : undefined}
       fit
       formModal={false}
     >
-      <Divider />
-      <Box bg="bg.0" h="100%">
-        {children({ embedType, goToNextStep, goBackToEmbedModal })}
-      </Box>
+      {!isEmbeddingSetupStage && (
```

Comment:
Can we keep the `EmbedTitle` here please? We've been back and forth on changing this between the milestones but it doesn't make sense to have the title either in the Modal or in the `EmbedModalHeader` considering the only thing that's changing is the string.